backend/src/backend/models.py

```diff
@@ -0,0 +1,104 @@
+import uuid
+from datetime import datetime, timedelta
+from typing import List
+
+from sqlalchemy import UUID, DateTime, ForeignKey, String
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy.sql import func
+from sqlalchemy.types import JSON
+from sqlalchemy_utils import EmailType, JSONType, UUIDType
+
+from backend.constants import COLLECTION_EXPIRE_TIME
+
+from .database import Base
+
+
+class User(Base):
+    """
+    User model, used for managing users.
+    """
+
+    __tablename__ = "users"
+
+    id: Mapped[UUID] = mapped_column(
+        UUIDType(binary=False), primary_key=True, default=uuid.uuid4, index=True
+    )
+    name: Mapped[str] = mapped_column(
+        String, unique=True, index=True, nullable=False, default=""
+    )
+    created_on = mapped_column(DateTime(timezone=True), default=datetime.utcnow())
```

Comment:
I remember we had difficulty with the serialization/deserialization of Datetime with postrges on zimfarm.
@benoit74 do you recall? Can you elaborate?